plottable.css

```diff
@@ -78,6 +78,14 @@ svg.plottable {
   fill: #32313F;
 }
 
+.plottable .off-bar-label text {
+  fill: #5279c7;
+}
+
+.plottable .on-bar-label text {
+  fill: white;
```

Comment:
Probably we should leave this class empty -- with this class, all on-bar labels are white, instead of color-dependent, which looks odd if the bars are light-colored:

<img width="415" alt="screen shot 2015-08-17 at 1 25 28 pm" src="https://cloud.githubusercontent.com/assets/1440449/9315428/01edcbbc-44e4-11e5-96d4-6d8a30951a48.png">